src/TTrigger.cpp

```diff
@@ -580,6 +580,8 @@ inline void TTrigger::updateMultistates(int regexNumber, std::list<std::string>&
         pCondition->multiCapturePosList.push_back(posList);
         if (nameMatches != nullptr) {
```

Comment:
IIRC this can be written more concisely as:
```suggestion
        if (nameMatches) {
```
and it does make the block of code read more sensibly (IMHO) in that "if the pointer is valid, use what it points at - otherwise *don't*!"). :man_shrugging:

Reply:
Sure 👍🏻